extensions/guacamole-auth-ldap/src/main/java/org/apache/guacamole/auth/ldap/LDAPGuacamoleProperties.java

```diff
@@ -153,4 +153,14 @@ private LDAPGuacamoleProperties() {}
 
     };
 
+    /**
+     * The behavior of alias dereferencing for the LDAP connections.
```

Comment:
But what does this mean?

It's clear to me since I've read what you've written in JIRA, etc., but the comment doesn't tell me what changing this setting does. I would have to dig through other parts of the code to find out that "alias dereferencing" is actually a consideration for queries against LDAP, but the comment as written implies this aliases have something to do with the LDAP connection itself.

In your other PR for the documentation covering this new property (apache/incubator-guacamole-manual#38), you say:

> Controls whether or not the LDAP connection follows (dereferences) aliases as it searches the tree.

Something like that would be perfect. Descriptive, accurate, yet not overly verbose.